korgan/pro_document_quality.py

```
@@ -0,0 +1,178 @@
+        "korgan_voice_response_to_claim",
+        # досудебная претензия и ответ на неё
+        "korgan_pretrial_demand",
+        "korgan_pretrial_response",
+        "korgan_10_of_10_pretrial",
+        "korgan_10_of_10_pretrial_response",
+        "korgan_voice_pretrial_response",
+        "korgan_uploaded_pretrial_claim_recovery",
+    }
+)
+
+# Лимиты вывода на составление. Иск с расчётом по периодам, разбором исковой
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)